acme_diags/viewer/default_viewer.py

```diff
@@ -73,22 +80,25 @@ def create_viewer(root_dir, parameters):
 
                     if parameter.plevs == []:  # 2d variables.
                         if parameter.run_type == 'model_vs_model':
-                            row_name = '{} {}'.format(var, region)
+                            row_name = '{} {}'.format(var_title, region)
                         else: 
-                            row_name = '{} {} {}'.format(var, region, ref_name)
+                            row_name = '{} {} {}'.format(var_title, region, ref_name)
                         fnm = '{}-{}-{}-{}'.format(ref_name,
-                                                    var, season, region)
+                                                    var_title, season, region)
+                        print('DEBUG: Row name A: {}'.format(row_name))
+                        # TODO: Come up with better row names
```

Comment:
What would be a better row name?
For reference, in the tests, the row name for `lat_lon_vectors` is `U-V-850mb global ERA-Interim` and the row name for `lat_lon` is `T-850mb global ERA-Interim`. Those row names seem more or less equivalent in what information they give.

Reply:
`U-V-850mb global ERA-Interim` is a good name